richie/settings.py

```diff
@@ -256,14 +257,36 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
     CMS_PERMISSION = True
 
     CMS_PLACEHOLDER_CONF = {
-        "courses/cms/course_detail.html course_video": {
+        # Course detail
+        "courses/cms/course_detail.html course_teaser": {
             "name": _("Course teaser"),
             "plugins": ["VideoPlayerPlugin"],
             "limits": {"VideoPlayerPlugin": 1},
         },
-        "courses/cms/course_detail.html course_syllabus": {  # all plugins allowed
-            "name": _("Course Syllabus")
+        "courses/cms/course_detail.html course_syllabus": {
+            "name": _("Course Syllabus"),
+            # Enforce Section plugin usage at first level
+            "plugins": ["SectionPlugin"],
```

Comment:
I think course syllabus should receive a CKEditor directly. The h2 title is already in the template.